packages/components/webpack.config.js

```diff
@@ -106,5 +106,10 @@ module.exports = {
         }),
         new IgnorePlugin(/^\.\/locale$/, /moment$/)
     ],
-    externals: ['react', 'react-dom', 'reactn', 'react-bootstrap', 'immutable', 'jquery']
+    externals: [
```

Comment:
nit: This would be slightly easier to read/follow if it was one item per line and alphabetical.

Reply:
Agreed